### ISSUE TITLE
Postcode Searching

### ISSUE THREAD

**Issue by tp-pbda** [ethercreative/simplemap#278]
### Description
Though it **WORKS** when doing it by adding more info to be explicit
`{% set location = craft.entries.maps({
		location: '32259 FL'
})  %}`

But **NOT WORKING** using **zip/post code** alone
`{% set location = craft.entries.maps({
		location: '32259'
})  %}`

### Additional info

- Craft version: 3.5.11.1
- Maps version: 3.8.4.1
- PHP version: 7.2
- Database driver & version: MySQL 5.7.19

**Comment by matthewstick** [comment]
Is this fixed yet?

**Comment by jamesmacwhite** [comment]
I don't think so. I tried searching UK postcode formats and it doesn't work with Google Maps. I ended up adding a custom search attribute `postcode` and getting the postcode value from the map parts data and allowing an entry with a map field to be searched using that attribute value, so basically create a search by postcode with the Craft search function.
